.pre-commit-config.yaml

```diff
@@ -1,25 +1,48 @@
+exclude: >
+  (?x)(
+      docs/|
+      node_modules/
+  )
 repos:
--   repo: https://github.com/ambv/black
-    rev: 22.3.0
-    hooks:
-    - id: black
-      language_version: python3
-      args:
-        - --line-length=79
-        - --experimental-string-processing
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.2
-    hooks:
-    - id: flake8
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.1.0
-    hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
-      - id: check-ast
--   repo: https://github.com/asottile/reorder_python_imports
-    rev: v2.6.0
-    hooks:
-      - id: reorder-python-imports
-        name: Reorder Python imports (src, tests)
-        args: ["--application-directories", "src"]
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.4.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-yaml
+- repo: https://github.com/myint/autoflake
+  rev: v2.1.1
+  hooks:
+  - id: autoflake
```

Comment:
Not sure where some of these pre-commit checks have come from?
Can we make it consistent with https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/main/.pre-commit-config.yaml please.With the addition of djLint for the jinja templates as per https://github.com/communitiesuk/funding-service-design-assessment/blob/f578cbaf4d8c5a2d6bb1360bc26ef067a7c9a580/.pre-commit-config.yaml#L26-L30 ?

Reply:
Changed to make consistent with data-store and added djLint for jinja templates.